Plugins/SimpleFileBrowser/Scripts/FileBrowser.cs

```diff
@@ -42,13 +42,42 @@ private struct QuickLink
 		#endregion
 
 		#region Inner Classes
-		public class Filter
+
+		public interface IFilter {
```

Comment:
Please get rid of the free space and put the curly bracket on a new line for consistency (I'm asking this since this is a small commit).